resources/views/themes/default1/order/show.blade.php

```diff
@@ -422,6 +413,12 @@
 
 @section('icheck')
 <script>
+  $(document).ready(function(){
+    let ele = document.getElementById("s_key").textContent;
+     let finalele = ele.match(/.{1,4}/g).join('-');
```

Comment:
this script is seen multiple times in this PR. We could have tried some way of reusing it

Reply:
Moved under common/js file